apps/sms/js/thread_list_ui.js

```diff
@@ -407,14 +456,17 @@ var ThreadListUI = {
   // Since removeThread will revoke list photoUrl at the end of deletion,
   // please make sure url will also be revoked if new delete api remove threads
   // without calling removeThread in the future.
-  delete: function thlui_delete() {
+  delete: function thlui_delete(selected) {
+    if (this.selectionHandler) {
+      selected = this.selectionHandler.selectedList;
+    }
```

Comment:
as said earlier: you need `if (selected)` instead of `if (this.selectionHandler)`